File: Level1/17.시저암호/17.시저암호.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

string solution(string s, int n) {
    string answer = "";
    for(int i = 0 ; i<s.size() ; i++){
        char element = s[i];
        if(element == ' '){
            continue;
        }else if(element>='a' && element<='z'){
            element += n;
            if(element>='a' && element<='z'){
                s[i] = element;
            }else{
                s[i] = 'a' + element - 'z' -1;
            }

        }else{
            element += n;
            if(element>='A' && element<='Z'){
                s[i] = element;
            }else{
                s[i] = 'A' + element - 'Z' -1;
            }
            
        }
    }
    answer = s;
    return answer;
}

File: Level1/17.시저암호/17.시저암호.py
def solution(s, n):
    answer = ''
    for element in s:
        if element == ' ':
            answer += ' '
        elif element>='a' and element<='z':
            element = chr(ord(element) + n)
            if element>='a' and element<='z':
                answer += element
            else:
                answer += chr( ord('a') + ord(element) - ord('z') -1 )
        else:
            element = chr(ord(element) + n)
            if element>='A' and element <='Z':
                answer += element
            else:
                answer += chr( ord('A') + ord(element) - ord('Z') -1 )
    return answer